frontend/apps/picasso/package.json

```diff
@@ -25,7 +25,8 @@
     "@mui/material": "^5.5.2",
     "@polkadot/api": "^8.3.2",
     "@polkadot/extension-dapp": "^0.42.7",
-    "@polkadot/util": "^10.1.6",
+    "@polkadot/util": "^9.4.1",
```

Comment:
This will create version warnings in the build, is there a specific reason that we need to downgrade?

Reply:
@easteregg not really, i was playing around with different method and might have installed this one, we will keep same version